<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrainDlgForm</class>
 <widget class="QWidget" name="TrainDlgForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>训练模式</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>初始资金:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>训练开始日:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="dbspbBegCapital">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <double>10000000.000000000000000</double>
   </property>
   <property name="value">
    <double>1000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtnBuy">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>买</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtnSell">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>卖</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtnNextK">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>下一根=&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtnPreK">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;=上一根</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>170</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>手续费:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="dbspbFeeRate">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>170</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="maximum">
    <double>99.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.001000000000000</double>
   </property>
   <property name="value">
    <double>0.025000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>60</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>611</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>成交记录</string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendar">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>0</y>
     <width>261</width>
     <height>151</height>
    </rect>
   </property>
   <property name="gridVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_date">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>19</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>158</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pbtnStart">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>开始</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pbtnStop">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>结束</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
